engine/table/python/core/deephaven_jpy_init.py

```diff
@@ -18,188 +16,5 @@
-        if not key.startswith("__"):
-            if hasattr(value, 'methods'):
-                methods = value.methods
-                if methods:
-                    if methods[0].is_static:
-                        globals()[value.name] = value
-
-    ql.importClass(_jclass_.forName(clazz))
-
-
-def java_array(type, values):
-    return jpy.array(type, values)
-
-
-def IntArray(values):
-    return java_array('int', values)
-
-
-def DoubleArray(values):
-    return java_array('double', values)
-
-
-def FloatArray(values):
-    return java_array('float', values)
-
-
-def LongArray(values):
-    return java_array('long', values)
-
-
-def ShortArray(values):
-    return java_array('short', values)
-
-
-def BooleanArray(values):
-    return java_array('boolean', values)
-
-
-def ByteArray(values):
-    return java_array('byte', values)
-
-
-def _exists_and_is_file_(prop):
-    """
-    For finding a fully qualified customer configured import file
-
-    :param prop: a stem of the import file
-    :return: the fully qualified file path
-    """
-
-    config = Config()
-    file_path = config.getStringWithDefault(prop, None)
-    if file_path:
-        file_path = file_path.replace("<devroot>", config.getDevRootPath())
-        if os.path.isfile(file_path):
-            return file_path
-
-    return None
-
-
-def _empty_or_comment_(line):
-    """helper method for extracting a line"""
-    return line is None or len(line.strip()) < 1 or line.strip().startswith("#")
-
-
-######################################################################################################
-# Perform the desired imports for the console workspace
-# NOTE: these can't be moved into a method or the imports would be out of scope for the console
-
-default_imports = _exists_and_is_file_("python.default.imports")
-if default_imports:
-    # test if line matches pattern "import <stuff>" or "from <stuff> import <stuff>"
-    import re
-
-    import_re = '^(?:import|(from(\s+)(.+))import)(\s+)(.+)'
-    import_pattern = re.compile(import_re)
-
-    with open(default_imports, 'r') as f:
-        for line in f:
-            # note that this pattern is repeated, but I want to avoid a method polluting the namespace
-            if _empty_or_comment_(line):
-                # this line is empty or a comment
-                continue
-
-            if ";" in line:
-                logging.error("Can not run line \n{}\n from python.default.imports, contains a ';'".format(line))
-                continue
-
-            try:
-                if import_pattern.match(line):
-                    exec(line)
-                else:
-                    logging.error("Could not run line \n{}\n from python.default.imports, "
-                                  "does not match import pattern" .format(line))
-            except ImportError:
-                logging.error("Could not import module: {}".format(line))
-    del import_re, import_pattern
```

Comment:
I'm not sure about the deletion of this section.  In Groovy, we have the ability to automatically exec some code at the start of a session, so that console users don't have to type as much.  In Python, we have backed away from that and have required users to be explicit.  Should we retain the functionality?  Maybe?  Maybe not.

Reply:
Doesn't application mode cover this use case, allowing the server to start up with some pre-defined script?